.github/workflows/test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         submodules: true
     - uses: actions/setup-node@v3
       with:
-        node-version: 16.x
+        node-version: 20.x
```

Comment:
Node 18 is still the current maintenance LTS version.  Is there a need to move to a version higher than 18?

Reply:
Oh, you're right. I wanted `Array.prototype.findLast`, but this is already present in 18.